api/python/provisioner/api.py

```diff
@@ -221,6 +221,37 @@ def set_swupdate_repo(
     )
 
 
+def set_swupgrade_repo(release, source=None, dry_run=False, nowait=False):
+    r"""
```

Comment:
Why do we need the r?
Also, do address the codacy notice.

Reply:
@dmitrydb please address the both things

Reply:
Done.

`r` is needed to show the one-line function description in help message for provisioner CLI